app/views/shared/buttons/_ab_list.html.haml

```diff
@@ -107,11 +107,11 @@
             %td.table-view-pf-select
               %i{:class => obj[:button_icon], :style => obj.key?(:button_color) ? "color: #{obj[:button_color]};" : nil}
             %td
-              = obj[:name]
+              = obj[:name] || obj["name"]
             %td
-              = obj[:description]
+              = obj[:description] || obj["description"]
 - elsif %w(ab_group_edit ab_group_new).include?(params[:pressed]) || (params[:button] == "reset" && params[:action] == "group_update")
-  = render :partial => 'shared/buttons/group_form'
+  = render :partial => 'shared/buttons/group_form'                      
```

Comment:
Please ensure your editor is configured to remove trailing whitespace to avoid changes like this.